### PR TITLE
Check license boilerplate for python files

### PR DESCRIPTION
**Pull request by eparis**
Check for license boilerplate in all python files (as well as .sh and .go)

### CODE REVIEW

**Comment by eparis** [comment]
This actually contains https://github.com/GoogleCloudPlatform/kubernetes/pull/7654  so should probably get merged after

**Comment by jlowdermilk** [comment]
LGTM. I think it needs a rebase tho.

File: hooks/boilerplate.py.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```
Comment (nikhiljindal) [inline hooks/boilerplate.py.txt]: Should this be 2015 instead of 2014?

Comment (eparis) [inline hooks/boilerplate.py.txt]: No 2014 is right...   We regex all 2015->2014 before we compare with the boilerplate

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hooks/boilerplate.py#L54

**Pull request merged**